data/src/main/java/com/microsoft/azure/kusto/data/auth/ConnectionStringBuilder.java

```diff
@@ -92,6 +94,10 @@ String getAccessToken() {
         return tokenProvider;
     }
 
+    boolean getUseDeviceAuth() {
```

Comment:
Per naming convention, getters that return a boolean should start with "is". I would go with "isUseDeviceCodeAuth"